src/pages/PreviewScreen.jsx

```diff
@@ -154,18 +154,6 @@ const PreviewScreen = ({ getUserToken }) => {
     );
   }
 
-  // This for handling beam recording
-  if (skipPreview) {
-    return loginInfo.token ? (
-      <Redirect to={`/meeting/${urlRoomId}/${userRole}`} />
```

Comment:
Redirect is taken care in join() call itself after fetching the token. Redundant here.